Source/WebCore/dom/EventNames.json

```diff
@@ -0,0 +1,321 @@
+[
+    {"name": "DOMActivate", "types": ["MouseClickRelated"]},
```

Comment:
No strong opinions, but it feels like this could be a key value object instead of an array, since the name is a required key ? Like `CSSProperties.json` or `CSSPseudoSelectors.json`.

Reply:
Sure, changed.